environment.devenv.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
   {% endif %}
   - sqlalchemy ==1.3.15  # [PY==36]
   - sqlalchemy ==1.4.44  # [PY==310]
-  - sqlalchemy-utc>=0.10
+  - sqlalchemy-utc>=0.14
```

Comment:
Should we restrict the use of older versions on serialchemy? @lvoliveira

Reply:
Only if serialchemy requires a feature that is only available on newer versions, and would brake with older ones. Is that the case? If yes, this is only one place where that change should be done, we would also need to change the `setup.py` file.

Reply:
Not really, I will close this PR then